packs/victorops/pack.yaml

```diff
@@ -0,0 +1,5 @@
+name: Victorops
+description: Packs which allows integration with Victorops events.
```

Comment:
Please also add keywords attribute.

See https://github.com/StackStorm/st2contrib/blob/master/packs/chef/pack.yaml#L4 for example.